lms/static/sass/course/instructor/_instructor_2.scss

```diff
@@ -1465,7 +1465,7 @@
       .metrics-overlay-content-wrapper {
         position: relative;
         display: block;
-        height: 475px;
+        height: 530px;
```

Comment:
So, upstream is affected but slightly more so that the change needs to be 550px.

However, the metrics tab doesn't work at all for upstream. I had to make code changes to get it working enough to reproduce. Not being sure what bag of worms this might open upstream I would have thought moving this code to theme files would be best.

Reply:
Hrm, it actually looks like that height declaration may be superfluous after all...

There's already an explicit height set to the inner contents (the part of the modal that's scrollable, `.metrics-overlay-content`) that would prevent the modal from growing too large.

Removing the height from the parent container seems like it allows the entire modal to resize as necessary. As the content grows (slightly), the modal follows (but again, shouldn't get too large, due to the height on the child div).

Given that, a fix that instead _removes_ the height instead of just bumping it, should future-proof this for us (in the event that this modal ever grows again), and may even fix this issue upstream too, despite the 20px difference (530 vs 550).

All said, a good course of action here seems to be:
- test removal of this height in an edx devstack (I have already tested in a Stanford stack locally)
- move this change to the theme and out of platform (until the upstream PR lands)
- update this change on our theme to instead use `height: auto`
  - note: Overriding `height` to `auto` is equivalent to removing the original declaration (which we'll remove w/ the upstream PR)
- _in_ _parallel_: PR the removal of this line upstream
  - this upstream PR can also serve as a request-for-comments, regarding the state of the metrics tab upstream (assuming it doesn't actually work for them)

Reply:
👍